### ISSUE TITLE
Taking screenshot using Flutter Web

### ISSUE THREAD

**Issue by israalfikiii** [media-kit/media-kit#1050]
Hello guys, 
I am using this package to view and play videos in my flutter web app. I want to capture a screen shot for the current moment in video whether it's playing or paused. I tried to do it by following the documentation but it keeps returning null. What could be the reason for it? What am I missing? 

Here's my code:
`  /// Upload image to server & attach it to given local answer
  static Future<void> uploadAnswerImage({
    required Player videoPlayer,
    required VideoController videoController,
    required int roomId,
    required int questionId,
  }) async {
    // take screen shot of current moment in video
    final Uint8List? bytes = await videoPlayer.screenshot(format: 'image/png');
    print('screenshot is ${bytes?.length}');
    if (bytes == null) return;

    // generate url for uploading image
    final file = FileModel(
      size: bytes.length,
      name: "screenshot_question$questionId",
      destination: "rooms/$roomId",
      fileBytes: bytes,
    );

    final generatedFile = await FileManager.generateUploadUrl(
      file: file,
      mimeType: "image/png",
    );

    print('generatedFile url is ${generatedFile?.url}');
  }`

**Comment by alexmercerind** [comment]
CORS

**Comment by israalfikiii** [comment]
What exactly can I do to solve it? so taking screenshots is not working with Flutter Web?

**Comment by israalfikiii** [comment]
I also used this package 'package:screenshot/screenshot.dart', I wrapped the video with a screenshot widget. It's working for all widgets including Text but when it comes to the video, it takes returns a black image.

**Comment by alexmercerind** [comment]
I don't think it can be fixed unless you change the CORS policy of the video that you're serving. Things are limited on the web.

**Comment by israalfikiii** [comment]
I solved it by forking the package and editing it. 
In the file: media-kit\media_kit\lib\src\player\web\player\real.dart
I added this line when setting the attribute for the video element. 
`        ..setAttribute('crossorigin', 'anonymous')
`
This doesn't trigger CORS and allows the screenshot to be taken.

**Comment by alexmercerind** [comment]
Can you send pull-request?

**Comment by khlebobul** [comment]
> I solved it by forking the package and editing it. In the file: media-kit\media_kit\lib\src\player\web\player\real.dart I added this line when setting the attribute for the video element. ` ..setAttribute('crossorigin', 'anonymous')` This doesn't trigger CORS and allows the screenshot to be taken.

It really worked, thanks! @israalfikiii can you send PR?
